web/src/main/java/org/mskcc/cbio/oncokb/api/pub/v1/AnnotationsApiController.java

```diff
@@ -44,7 +44,7 @@ public class AnnotationsApiController {
     @RequestMapping(value = "/annotate/mutations/byProteinChange",
         produces = {"application/json"},
         method = RequestMethod.GET)
-    public ResponseEntity<IndicatorQueryResp> annotateMutationsByProteinChangeGet(
+    public ResponseEntity<Object> annotateMutationsByProteinChangeGet(
```

Comment:
We can't change this. The API client will not be able to generate expected response body.

Reply:
Doesn't this existing attribute take care of that problem?

```java
@ApiResponse(code = 400, message = "Error, error message will be given.", response = String.class)})
```